lib/bolt/pal.rb

```diff
@@ -357,6 +357,39 @@ def get_task(task_name)
       Bolt::Task.from_task_signature(task)
     end
 
+    def list_plans_with_cache(filter_content: false)
+      # Don't filter content yet, so that if users update their plan filters
+      # we don't need to refresh the cache
+      plan_names = list_plans(filter_content: false).map(&:first)
+      plan_cache = if @project
+                     Bolt::Util.read_optional_json_file(@project.plan_cache_file, 'Plan cache file')
+                   else
+                     {}
+                   end
+      updated = false
+
+      plan_list = plan_names.each_with_object([]) do |plan_name, pl|
```

Comment:
```suggestion
      plan_list = plan_names.each_with_object([]) do |plan_name, list|
```

Nitpick, but it wasn't readily apparent what `pl` was (though that's also partially to just coming back from break :P)